src/Symfony/Component/Messenger/Bridge/AmqpExt/composer.json

```diff
@@ -0,0 +1,41 @@
+{
+    "name": "symfony/messenger-amqp",
+    "type": "symfony-bridge",
+    "description": "Symfony AMQP extension Messenger Bridge",
+    "keywords": [],
+    "homepage": "https://symfony.com",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Fabien Potencier",
+            "email": "fabien@symfony.com"
+        },
+        {
+            "name": "Symfony Community",
+            "homepage": "https://symfony.com/contributors"
+        }
+    ],
+    "require": {
+        "php": "^7.2.5",
+        "ext-amqp": "*",
```

Comment:
I get why the ext is required here. But the extension `amsp` wasn't required in `symfony/messenger`. Given the bridge is a requirement for messenger, now `symfony/messenger` requires amqp+redis+doctrine, which may not be really needed

same for ext-redis.

Reply:
Wow. You are correct!

Great catch. 
They should be added in 6.0. 

I’m on my phone now. Will update later tonight

Reply:
This is resolved.